backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -706,27 +706,33 @@ internal class CodeGeneratorVisitor(val context: Context, val lifetimes: Map<IrE
                 || declaration.isExternal
                 || body == null)
             return
-
-        generateFunction(codegen, declaration,
-                declaration.location(start = true),
-                declaration.location(start = false)) {
-            using(FunctionScope(declaration, it)) {
-                val parameterScope = ParameterScope(declaration, functionGenerationContext)
-                using(parameterScope) usingParameterScope@{
-                    using(VariableScope()) usingVariableScope@{
-                        recordCoverage(body)
-                        if (declaration.isReifiedInline) {
-                            callDirect(context.ir.symbols.throwIllegalStateExceptionWithMessage.owner,
-                                    listOf(context.llvm.staticData.kotlinStringLiteral(
-                                            "unsupported call of reified inlined function `${declaration.fqNameForIrSerialization}`").llvm),
-                                    Lifetime.IRRELEVANT)
-                            return@usingVariableScope
-                        }
-                        when (body) {
-                            is IrBlockBody -> body.statements.forEach { generateStatement(it) }
-                            is IrExpressionBody -> generateStatement(body.expression)
-                            is IrSyntheticBody -> throw AssertionError("Synthetic body ${body.kind} has not been lowered")
-                            else -> TODO(ir2string(body))
+        val isLocalFunction = declaration.origin == IrDeclarationOrigin.LOCAL_FUNCTION_FOR_LAMBDA
```

Comment:
So, only inline lambdas can reference other file?

Reply:
If it referenced from this file we just do additional switch to same file. perhaps it better to avoid of course, but don't break anything.

Reply:
So, may be the name for the variable should be `isInlineLambda`?

Reply:
i could be misleading because it's lmbda but not inlined one. So maybe `isNotInlinedLambda`?